packages/terra-table/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "terra-table",
+  "main": "lib/index.js",
+  "version": "4.36.0",
+  "description": "The terra-table component provides user a way to render selectable data in a tabular format",
```

Comment:
Should this be "The Terra Table component ..."

Reply:
Updated the description.